Watchman.Discord/Areas/Responses/Services/ResponseCleanupService.cs

```diff
@@ -0,0 +1,37 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Watchman.DomainModel.Responses;
+
+namespace Watchman.Discord.Areas.Responses.Services
+{
+    public class ResponseCleanupService
+    {
+        private readonly ResponsesGetterService _responsesGetterService;
+        private readonly ResponsesService _responsesService;
+
+        public ResponseCleanupService(ResponsesGetterService responsesGetterService, ResponsesService responsesService)
+        {
+            _responsesGetterService = responsesGetterService;
+            _responsesService = responsesService;
+        }
+
+        public async Task CleanDuplicatedResponses()
+        {
+            var allResponses = this._responsesGetterService.GetResponsesFromBase().ToList();
+            var defaultResponses = allResponses.Where(resp => resp.ServerId == 0);
+            var serverResponses = allResponses.Where(resp => resp.ServerId != 0);
+            var responsesToDelete = serverResponses
+                .Where(serverResponse => defaultResponses.Any(defaultResponse => IsTheSame(serverResponse, defaultResponse)));
+
+            foreach (var response in responsesToDelete)
+            {
+                await this._responsesService.RemoveResponse(response.OnEvent, response.ServerId);
+            }
+        }
+
+        private static bool IsTheSame(Response serverResponse, Response defaultResponse) {
```

Comment:
nazwałbym to bardziej jako ResponsesAreEqual
Albo CompareResponses 

IsTheSame teoretycznie nie mówi
ale jest wewnątrz `ResponseCleanupService` więc teoretycznie wiadomo że może chodzić o obiekt `Response`

Reply:
Zostawie może samo Compare(Response serverResponse, Response defaultResponse)  ?